style.css

```diff
@@ -0,0 +1,146 @@
+.main {
+  visibility:hidden;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+}
+
+.dash {
+  margin: 0 15px;
+  width: 35px;
+  height: 15px;
+  border-radius: 8px;
+  background: #FF2CBD;
```

Comment:
Codacy found an issue: [Expected "#FF2CBD" to be "#ff2cbd" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/jstask35/pullRequest?prid=6109934)